### ISSUE TITLE
Frame extraction failed

### ISSUE THREAD

**Issue by shining99armor** [n00mkrad/flowframes#330]
![2695a6f3e0700333](https://github.com/user-attachments/assets/c63ba262-66a7-4d7c-a750-6583af3cb719)

After clicking the interpolate button, the following error appears. I tried to fix it somehow, but nothing helps. 
Everything worked on Windows 11, I reinstalled it on Windows 10 and it stopped working.

**Comment by Ammarmughal1** [comment]
Yeah, the same problem has occurred for me on Windows 11. I guess it’s not a Windows related error.

**Comment by AlphaIsNotBeta** [comment]
I'm not sure why did your happened. But for me, I have the same issues when the source video is encoded with FFV1.

**Comment by Pentazer** [comment]
This is also happening for me on animated gifs, regardless of source. tried it on RIFE 1.8, 3.0, and 4.0. It says incompatible but I've tried multiple different gifs from different places and it gives the same error on all of them, so I don't think it's the files.

**Comment by TigreChat** [comment]
I have used it successfully with JPG files as input (saved from "DImage viewer"). I have Windows 11 - so it isn't a Windows issue.
But I want to run it again on the newly created JPG files, but it doesn't recognize the files it wrote itself!
I run them through XNconvert and **sometimes** it accepts them - sometimes not. It **sometimes** accepts files written by PhotoShop (but not always) and when there are more than a dozen it isn't practical anyway (no batch processing).
I have tried using different JPG parameters but it isn't consistent. Have also tried converting the files to PNG, but it doesn't work either.
All of these files can be opened in PhotoShop or PolyView with no problems.
So I think there is definitely an internal problem which causes this error.
